src/MercadoPago/Resources/Payment/Card.php

```diff
@@ -41,6 +41,9 @@ class Card
         "cardholder" => "MercadoPago\Resources\Payment\Cardholder"
     ];
 
+    public ?int $bin;
```

Comment:
[req] Adicionar comentário e corrigir tipagem do campo bin, ele é string.
```suggestion
    /** Card bin. */
    public ?string $bin;
```

Reply:
Ajustado.